tests/dummy/app/templates/ember-flexberry-dummy-suggestion-edit.hbs

```diff
@@ -131,17 +132,15 @@
       limitFunction=null
       class=(if model.errors.author "error" "")
     }}
+    {{flexberry-validationmessage error=(t "forms.ember-flexberry-dummy-suggestion-edit.vauthor-caption") pointing="pointing"}}
   </div>
-  <div class="field">
+  <div class="field {{if model.errors.editor1 "error" ""}}">
     <label>{{t "forms.ember-flexberry-dummy-suggestion-edit.editor-caption"}}</label>
-    {{#if model.errors.editor}}
-      <span style="color:red">{{model.errors.editor}}</span>
-    {{/if}}
     {{flexberry-lookup
       value=model.editor1
       projection="ApplicationUserL"
       displayAttributeName="name"
-      title="Editor"
+      title="Editor1"
```

Comment:
This name (Editor1) was given in the diagram in the model. So we had to use it for validation. Otherwise, validation is not included.